crates/rerun/src/web_viewer.rs

```diff
@@ -4,54 +4,76 @@ use re_log_types::LogMsg;
 /// * A web-server, serving the web-viewer
 /// * A `WebSocket` server, server [`LogMsg`]es to remote viewer(s).
 struct RemoteViewerServer {
-    web_server_join_handle: tokio::task::JoinHandle<()>,
     sender: re_smart_channel::Sender<LogMsg>,
+    shutdown_tx: tokio::sync::broadcast::Sender<()>,
 }
 
 impl Drop for RemoteViewerServer {
     fn drop(&mut self) {
         re_log::info!("Shutting down web server.");
-        self.web_server_join_handle.abort();
+        self.shutdown_tx.send(()).unwrap();
```

Comment:
```suggestion
        self.shutdown_tx.send(()).ok();
```

panicing in drop is really bad